jib-core/src/main/java/com/google/cloud/tools/jib/image/ReproducibleLayerBuilder.java

```diff
@@ -37,6 +41,11 @@
  */
 public class ReproducibleLayerBuilder {
 
+  private static final ImmutableSet<PosixFilePermission> DEFAULT_FILE_PERMISSIONS =
```

Comment:
Leave these as integers — there's no point continuously switching between integer representations to non-integer representations.